### ISSUE TITLE
[MDB] Mob difference isn't highlighted

### ISSUE THREAD

**Issue by CheffieGithub** [spacestation13/BYONDDiffBots#19]
### Description

When comparing diff between map changes, mobs aren't highlighted in red.

### Steps to reproduce

1. make a mapping change that deals with mobs
2. view the diff
3. cry when you can't tell what mobs are changed

### Expected Behavior

All diff is highlighted for ease of viewing and review

### Additional Information & Screenshots

![image](https://user-images.githubusercontent.com/113442598/223397126-0a489a5f-4fa6-4b2d-926d-0604290bf5ac.png)

Case of object critter to mob critter

**Comment by CheffieGithub** [comment]
Yes that's the issue because they aren't the same, left the critter is an object on the right its a mob

**Comment by AffectedArc07** [comment]
The diff image can only compare visual changes, the exact same issue occurs for VV edits. 

---

Its unlikely this will ever be fixed, paging @MCHSL and @jupyterkat for thoughts.

**Comment by jupyterkat** [comment]
this would be a huge rework of how the diff is rendered, all for a tiny vv edit

**Comment by CheffieGithub** [comment]
well thats too bad then
